src/gt4py/next/iterator/transforms/fuse_as_fieldop.py

```diff
@@ -198,7 +198,10 @@ def _arg_inline_predicate(node: itir.Expr, shifts: set[tuple[itir.OffsetLiteral,
     ) or cpm.is_call_to(node, "if_"):
         # always inline arg if it is an applied fieldop with only a single arg
         if is_applied_fieldop and len(node.args) == 1:
-            return True
+            # ... and not too much overcompute
+            # TODO(havogt): tune this value or improve heuristic
```

Comment:
leave 4xlap and this problem for later, but use a itir test for the temporary problem